kubernetes/utils/create_from_yaml.py

```diff
@@ -30,7 +30,7 @@ def create_from_yaml(
     Perform an action from a yaml file. Pass True for verbose to
     print confirmation information.
     Input:
-    yaml_file: string. Contains the path to yaml file.
+    yaml_file: string. Contains yaml string or a  path to yaml file.
```

Comment:
Is it just me or there are two spaces between "a" and "path"?

Reply:
No, it's not just you. You are correct.

Reply:
This is now fixed.